remill/Arch/AArch64/Arch.cpp

```diff
@@ -1689,6 +1689,15 @@ bool TryDecodeAND_64_LOG_SHIFT(const InstData &data, Instruction &inst) {
   return TryDecodeEOR_64_LOG_SHIFT(data, inst);
 }
 
+// ANDS  <Wd>, <Wn>, #<imm>
+bool TryDecodeANDS_32S_LOG_IMM(const InstData &data, Instruction &inst) {
+  return TryDecodeEOR_32_LOG_IMM(data, inst);
```

Comment:
This is not quite correct. `EOR` is `EOR  <Xd|SP>, <Xn>, #<imm>`, so the if `Rd` is 31, then it's treated as `SP`/`WSP`, whereas with `ANDS`, it's treated as `XZR`/`WZR`.